schema.graphql

```diff
@@ -66,17 +79,13 @@ type StrategyReportResult @entity {
   startTimestamp: BigInt!
   endTimestamp: BigInt!
   "The duration (in days) from the previous report."
-  duration: BigDecimal!
+  duration: BigInt!
   "Duration percentage rate."
   durationPr: BigDecimal!
```

Comment:
i left durationPr as a BigDecimal because the value is so small that leaving it as BigInt would always show 0